.github/workflows/lint-format.yml

```diff
@@ -45,7 +45,7 @@ jobs:
       - name: Generate diff
         run: git diff HEAD > wpiformat-fixes.patch
         if: ${{ failure() }}
-      - uses: actions/upload-artifact@v3
+      - uses: actions/upload-artifact@v4
```

Comment:
This must be in sync with download-artifact versions as well, I believe.

Reply:
https://github.com/spacey-sooty/photonvision/blob/actions-versions/.github/workflows/build.yml#L235 it already appears to be on v4

Reply:
Horrifying. How did this code ever work.

Reply:
:3